Local ctest on MI300X VF failures not related to this PR:
```
FAILED tests/test_analyze_commands.py::test_col_2 - TypeError: argument of ty...
FAILED tests/test_analyze_commands.py::test_col_3 - TypeError: argument of ty...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_5 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_6 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_baseline - TypeError: argument of...
FAILED tests/test_analyze_commands.py::test_dependency_MI100 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_filter_combinations_coverage - Ty...
FAILED tests/test_analyze_commands.py::test_g - TypeError: argument of type '...
FAILED tests/test_analyze_commands.py::test_missing_files_scenarios - TypeErr...
FAILED tests/test_analyze_commands.py::test_save_dfs - TypeError: argument of...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_0 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_1 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_2 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_3 - TypeError: arg...
FAILED tests/test_analyze_commands.py::test_kernel_verbose_4 - TypeError: arg..
```.

--- a/src/utils/tty.py
+++ b/src/utils/tty.py
@@ -182,7 +182,7 @@ def show_all(args, runs, archConfigs, output, profiling_config, roof_plot=None):
                             .loc[lambda d: d["Metric"].isin(common_metrics)]
                         )
                         if initial_index is None:
-                            initial_index= runs[key].dfs[table_config["id"]].index
+                            initial_index = runs[key].dfs[table_config["id"]].index
                         else:
                             runs[key].dfs[table_config["id"]].index = initial_index
 

